### ISSUE TITLE
Where does the Android app save downloaded articles

### ISSUE THREAD

**Issue by g33kdad** [wallabag/android-app#914]
# Issue details

### Duplicate?
Have you searched the issues of this repository if your issue is already known? yes

It's a stupid question, I'm sure, but where the f*** does the Android app download files when you choose to download from within the app?

### Actual behaviour
- Tell us what happens

### Expected behaviour
- Tell us what should happen

# Steps to reproduce the issue
1. 
2.
3. 

# Environment details
- **wallabag app version**: 
- **wallabag app installation source** (e.g. Gplay, F-Droid, manual): 
- **Android OS version**: 
- **Android ROM** (e.g. stock, LineageOS, SlimRom,…): 
- **Android hardware**: 
- **wallabag server version**: 
- **Do you have Two-Factor-Authentication enabled?**: yes/no

# Logs
## wallabag server
Please paste relevant wallabag server logs here (from file `<wallabag>/var/logs/prod.log`):

## Web server
Please paste relevant web server logs (e.g. from nginx, Apache, …) here:

**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here!

# Your experience with wallabag Android app
Have you had any luck using wallabag Android app before? (Sometimes we get tired of reading bug reports all day and a lil' positive end note does wonders)

**Comment by di72nn** [comment]
The app downloads files to the only location available without storage permission: its directory on external storage: `<external storage>/Android/data/fr.gaulupeau.apps.InThePoche/files/`.

**Comment by g33kdad** [comment]
Hi,
I don't have external storage...  So

M

**Comment by Maxiride** [comment]
@g33kdad 
Even with internal storage only android (as far as I know) emulates it as external.

You should have a path like /storage/emulated/0/...


_Sent from my MI 8 Lite using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by di72nn** [comment]
Yes you do.

TL;DR: you may not have external *memory*, but you have "external storage": `/sdcard/` or something like `/storage/emulated/0/`.

Longer version:
It's an Android concept that's there probably due to legacy reasons.
There's "internal storage" to which you as a user have no direct access (apps can store their private data in their private directories there) and there is "external storage" which is commonly in the internal *memory*, but has path like `/storage/emulated/0/` (on my Android 9 / LineageOS 16 it's a bunch of symlinks: `/sdcard -> /storage/self/primary -> /mnt/user/0/primary -> /storage/emulated/0`).

I see @Maxiride just answered that, but since I've already finished typing...

**Comment by g33kdad** [comment]
Thanks!

I found it. I really appreciate it, fellas

M
